apache/trafficcontrol
# Missing ProfileId during server creation returns Bad Request error

## What does this PR (Pull Request) do?


- [x] This PR fixes #4979 


## Which Traffic Control components are affected by this PR?


- Traffic Ops

## What is the best way to verify this PR?

1. Run Traffic_Ops locally
2. Run POST call to https://TO's-url:port/api/3.0/servers/ with a missing profileId or profile, or profileDesc
Note: You should the following response
```
{"alerts":
    [{
        "text":"'profileId' is required",
        "level":"error"}
    ]}
}
```

## If this is a bug fix, what versions of Traffic Control are affected?


- master (d1a50b0)

## The following criteria are ALL met by this PR




## Additional Information


Documentation has the correct example (a successful POST)
